homeworks/06-advanced-redux/src/App.tsx

```diff
@@ -66,5 +80,8 @@ function App() {
     </div>
   );
 }
+const mapStateToProps = (state: ShopState) => ({ state });
+
+const AppConnected = connect(mapStateToProps, {})(App);
```

Comment:
there can be just 
```
const AppConnected = connect(mapStateToProps)(App);
```

Reply:
removed connect, using hooks, see https://github.com/vbarbasins/evo-ts-bootcamp/pull/7/commits/40000cf19231ba1b304ed01ab3879930296ac2a9